modular_chomp/code/modules/mob/living/simple_mob/subtypes/vore/plants/pitcher.dm

```diff
@@ -59,61 +61,70 @@ GLOBAL_LIST_INIT(pitcher_plant_lure_messages, list(
 	var/scent_strength = 5 //How much can a hungry pitcher confuse nearby people?
 	var/last_lifechecks = 0 //Timing variable to limit vore/hungry proc calls
 	var/list/pitcher_plant_lure_messages = null
-
-	can_be_drop_prey = FALSE //CHOMP Add
-
-
-
+	can_be_drop_prey = FALSE
 
 /mob/living/simple_mob/vore/pitcher_plant //Putting vore variables separately because apparently that's tradition.
 	vore_bump_chance = 100
 	vore_bump_emote = "slurps up" //Not really a good way to make the grammar work with a passive vore plant.
 	vore_active = 1
 	vore_icons = 1
 	vore_capacity = 1
-	vore_pounce_chance = 0 //Plants only eat people who stumble into them.
+	vore_pounce_chance = 100 // Either this makes mobs get eaten for attacking it or nothing happens and I don't know which.
```

Comment:
Not sure this will help but I noticed an issue where pitcher plants keep getting murdered on Thor instead of eating things that try to attack it. Fixing that is something I try here, but if it doesn't work, whatever, I'll fix it in another PR.